app/domain/authentication/authn_jwt/consts.rb

```diff
@@ -21,8 +21,7 @@ module AuthnJwt
     NBF_CLAIM_NAME = "nbf"
     IAT_CLAIM_NAME = "iat"
     RSA_ALGORITHMS = %w[RS256 RS384 RS512].freeze
-    ECDSA_ALGORITHMS = %w[ES256 ES384 ES512].freeze
-    SUPPORTED_ALGORITHMS = (RSA_ALGORITHMS + ECDSA_ALGORITHMS)
+    SUPPORTED_ALGORITHMS = RSA_ALGORITHMS
```

Comment:
Its for future implementation ? Otherwise why having the same for RSA_ALGORITHMS?

Reply:
I can to suppress `RSA_ALGORITHMS`...